XenAdmin/RDP/AxMSTSCLib.cs

```diff
@@ -1,68 +1,7 @@
-/* Copyright (c) Citrix Systems, Inc. 
- * All rights reserved. 
- * 
- * Redistribution and use in source and binary forms, 
- * with or without modification, are permitted provided 
- * that the following conditions are met: 
- * 
- * *   Redistributions of source code must retain the above 
- *     copyright notice, this list of conditions and the 
- *     following disclaimer. 
- * *   Redistributions in binary form must reproduce the above 
- *     copyright notice, this list of conditions and the 
- *     following disclaimer in the documentation and/or other 
- *     materials provided with the distribution. 
- * 
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND 
- * CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, 
- * INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF 
- * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE 
- * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR 
- * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, 
- * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
- * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR 
- * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS 
- * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, 
- * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING 
- * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE 
- * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF 
- * SUCH DAMAGE.
- */
-
-/*
- * To create this file, I did the following:
- *
- * Make a directory.
- * 
- * Copy mstscax.dll version 5 and 6 into it as mstscax5.dll and mstscax6.dll.
- * The former comes from XP SP2, the latter from XP SP3 or Vista.
- * I used 5.1.2600.2180 and 6.0.6001.18000.
- * 
- * In PowerShell:
- * 
- * PS> $ax = 'C:\Program Files\Microsoft Visual Studio 8\SDK\v2.0\bin\AxImp.exe'
- * PS> & $ax mstscax6.dll /source
- * 
- * Copy the resulting MSTSCLib.dll and AxMSTSCLib.cs into XenAdmin/RDP.
- * 
- * PS> & $ax mstscax5.dll /source
- * 
- * Now, take the UUID out of the version 5 AxMSTSCLib.cs from the ClsidAttribute and constructor of
- * AxMsRdpClient2, and cut-and-paste it into the same place in the version 6 AxMSTSCLib.cs that you
- * put into XenAdmin/RDP.
- * 
- * Finally, remove the AssemblyVersion and TypeLibraryTimeStamp attributes from the top of that file.
- * 
- * You should now have this file, and a corresponding MSTSCLib.dll.  The idea is to have an interop
- * wrapper that looks for the version 6 class whenever you ask for AxMsRdpClient6, but for the version 5
- * class when you ask for AxMsRdpClient2.  This is then used by XSVNCScreen.cs to fall back through
- * the DLL versions.
- */
-
 //------------------------------------------------------------------------------

```

Comment:
We need to retain the Copyright notice.
Also, it would be nice to add a note on how this file has been generated